### PR TITLE
adding text image 100

### PR DESCRIPTION
**Pull request by JasmineDavidson94**
Created a new file and completed text on image 100. Thank you

### CODE REVIEW

**Comment by NtemKenyor** [comment]
This is a great work you have done. However, U have also sent to this project hidden files. Please check your PC and remove the hidden files(Files that start with a dot). Please do just that and I would merger your PR immediately. thank you.

**Review by NtemKenyor — CHANGES_REQUESTED** [review]
remove hidden files created by your PC.

**Comment by JasmineDavidson94** [comment]
Apologies for the delay, I did not realize the PR hadn't been accepted. Thank you for the feedback! I have tried making the changes but cannot locate those hidden files. Any suggestions on how to resolve this issue.

**Comment by JasmineDavidson94** [comment]
I have created a new branch and deleted the requested files there. A new pull request was made. If this fixes the issue I can close this request

File: modules/text_on_image/text_on_image_100_Jasmine_Davidson.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<div class="image-container">
+    <img src="https://images01.nicepagecdn.com/page/55/71/html-template-5571820.jpg" alt="Background Image">
```
Comment (NtemKenyor) [inline modules/text_on_image/text_on_image_100_Jasmine_Davidson.html]: You used the link of the image you were suppose to design.

**Review by NtemKenyor — COMMENTED** [review]
Great Job n and check the possible changes.

File: modules/text_on_image/text_on_image_100_Jasmine_Davidson.html (hunk 1 above)
Comment (NtemKenyor) [reply modules/text_on_image/text_on_image_100_Jasmine_Davidson.html]: your background image is really not a background image as it contains other text and write up...

**Review by NtemKenyor — APPROVED** [review]
Congratulations 👏

I would handle any other corrections myself. Thank you for contributing to Open Source development.

**Pull request merged**